docker/runner.go

```diff
@@ -7,36 +7,36 @@ import (
 	"os/exec"
 )
 
-// Runner of functions using the appsody binary.
+// Runner of functions using the docker command.
 type Runner struct {
 	// Verbose logging flag.
 	Verbose bool
 }
 
-// NewRunner creates an instance of an appsody-backed runner.
+// NewRunner creates an instance of a docker-backed runner.
 func NewRunner() *Runner {
 	return &Runner{}
 }
 
 // Run the function at path
 func (n *Runner) Run(path string) error {
-	// Check for the appsody binary explicitly so that we can return
+	// Check for the docker binary explicitly so that we can return
 	// an extra-friendly error message.
-	_, err := exec.LookPath("appsody")
+	_, err := exec.LookPath("docker")
 	if err != nil {
-		return errors.New("please install 'appsody'")
+		return errors.New("please install 'docker'")
 	}
 
-	// Extra arguments to appsody
+	// Extra arguments to docker
 	args := []string{"run"}
 
 	// If verbosity is enabled, pass along as an environment variable to the function.
 	if n.Verbose {
-		args = append(args, []string{"--docker-options", "-e VERBOSE=true"}...)
+		args = append(args, []string{"-e VERBOSE=true"}...)
```

Comment:
Should this also have `-rm -it -p 8080:8080`?

Reply:
Yes.  I believe the interactive flag -i won't be allowed, but the others yes.  This has been updated in the latest version.